docs/source/rbac.rst

```diff
@@ -232,18 +224,13 @@ There are some exceptions, described below:
 Enabling RBAC
 -------------
 
-If you installed |ewc| using the :doc:`one-line install </install/ewc>`, RBAC will be automatically
-enabled. It will assign the ``admin`` role to ``stanley`` and ``st2admin``. 
-
-If you installed |ewc| separately, by installing the ``bwc-enterprise`` package on top of |st2|, you
-will need to manually enable RBAC, and assign ``admin`` privileges to ``stanley``. It is not
-enabled by default. To enable it, add this section to ``/etc/st2/st2.conf``:
+To enable rbac, add this section to ``/etc/st2/st2.conf``:
```

Comment:
Yeah, let's be a little bit more explicit that these configuration steps should be done manually. I think that should cover the comment by @amanda11.

```suggestion
To make RBAC work you will need to manually enable it and assign ``admin`` privileges to default user ``stanley``.

To enable RBAC, add this section to ``/etc/st2/st2.conf``:
```